src/function/scalar/strftime_format.cpp

```diff
@@ -733,7 +731,33 @@ bool StrpTimeFormat::Parse(string_t str, ParseResult &result) const {
 		data++;
 		size--;
 	}
+
+	//	Check for specials
 	idx_t pos = 0;
+	result.is_special = false;
+	if (Date::TryConvertDateSpecial(data, size, pos, Date::PINF)) {
```

Comment:
This seems to reduce performance of `benchmark/micro/cast/strptime.benchmark` by around 10~15%, perhaps we can be a bit more clever in when we take this branch (e.g. check if the first character is a character?)

Reply:
The benchmark is too fast for consistent results but profiling with Instruments showed a definite hot spot there.  Checking for size and the first character being alpha or minus removed the hot spot.

Reply:
Great! When I ran it I increased the number of rows generated in the benchmark to double-check that it was not just noise